ci.sh

```diff
@@ -2,6 +2,28 @@
 set -e
 
 # Please run it from root project directory
+
 # For some reason test for annotation processor are failing on a regular CI setup.
 # So we had to exclude test task for it from the main build process and execute it as a separate command.
-./gradlew clean build checkstyle -PdisablePreDex -x :storio-sqlite-annotations-processor-test:test -x :storio-content-resolver-annotations-processor-test:test && ./gradlew :storio-sqlite-annotations-processor-test:testDebugUnitTest && ./gradlew :storio-content-resolver-annotations-processor-test:testDebugUnitTest
+./gradlew clean build checkstyle -PdisablePreDex -x :storio-sqlite-annotations-processor-test:test -x :storio-content-resolver-annotations-processor-test:test
+./gradlew :storio-sqlite-annotations-processor-test:testDebugUnitTest
+./gradlew :storio-content-resolver-annotations-processor-test:testDebugUnitTest
+
+if git describe --exact-match --tags $(git log -n1 --pretty='%h') ; then
+    echo "Git tag detected, launching release process..."
+
+    if [ -z "$GPG_SECRET_KEYS" ]; then
```

Comment:
During code review we all need to make sure that nobody tries to print our environment variables, because that way we'll effectively leak our credentials and allow others to sign and upload jars from our account.